aries-site/src/examples/templates/dashboards/components/UpdateItem.js

```diff
@@ -0,0 +1,70 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { Anchor, Box, Button, Text } from 'grommet';
+import { FormDown, FormUp } from 'grommet-icons';
+
+export const UpdateItem = ({ title, description }) => {
+  // HPE Design System guidance for string length prior to truncation.
+  const MAX_STRING_LENGTH = 192;
+
+  const [showAllDescription, setShowAllDescription] = useState(false);
+  return (
+    <Box>
+      <Text weight="bold">{`${title} is available.`}</Text>
+      <Box gap="xsmall">
+        {description.length <= MAX_STRING_LENGTH ? (
+          <Text>{description}</Text>
+        ) : (
+          <>
+            {!showAllDescription ? (
+              <Text>{`${description.substring(
```

Comment:
Should this be a paragraph?

```suggestion
              <Paragraph>{`${description.substring(
```